OneSignalSDK/onesignal/src/main/java/com/onesignal/outcomes/model/OSOutcomeEvent.java

```diff
@@ -49,38 +68,25 @@ public float getWeight() {
         return weight;
     }
 
-    public JSONObject toJSONObject() {
+    public JSONObject toJSONObject() throws JSONException {
```

Comment:
This forces a try catch outside of this method why not just leave the try-catch internal so we dont need to expose it

Reply:
Because technically with the new format, architecture, models will only beholders of data, any exception will be handle by the manager, controller, etc using this method. And I wanted to abstract the needing of a logger on a model class too